nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java

```diff
@@ -227,7 +228,9 @@ public Object getFile(@HeaderParam(MAX_EXPECTED_SIZE) String maxExpectedSizeHead
             throw new PreConditionFailedException();
         }
 
-        logResponse(OPERATION_GET_FILE, OK.getStatusCode());
+        String itemVersion = blob.getDigest();
+        response.addHeader(ITEM_VERSION, itemVersion);
+        logResponse(OPERATION_GET_FILE, OK.getStatusCode(), ITEM_VERSION, itemVersion);
```

Comment:
Can't use `buildItemVersionResponse` here?

Reply:
Nop, we need to return the `blob` here, not am empty 200 response.

Reply:
I will use `buildItemVersionResponse` and ignore its return value.